index.html

```diff
@@ -4,10 +4,95 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link href="./style.css" rel="stylesheet">
     <title>Document</title>
 </head>
 
 <body>
+    <header>
+        <div class="navbar">
+            <nav>
+                <img class="logo" src="./img/Takeoff.png">
+                <div class="menu">
+                    <p><a class="tag-menu" href="#">OVERVIEW</a></p>
+                    <p><a class="tag-menu" href="#">HOW IT WORKS</a></p>
```

Comment:
It's semantically incorrect to use p here. Change tag to a different one.

Reply:
I removed p and left just a.
That's fixed!